docs/zh/14-reference/13-schemaless/13-schemaless.md

```diff
@@ -70,6 +70,7 @@ st,t1=3,t2=4,t3=t3 c1=3i64,c3="passit",c2=false,c4=4f64 1626006833639000000
 
 需要注意的是，这里的 tag_key1, tag_key2 并不是用户输入的标签的原始顺序，而是使用了标签名称按照字符串升序排列后的结果。所以，tag_key1 并不是在行协议中输入的第一个标签。
 排列完成以后计算该字符串的 MD5 散列值 "md5_val"。然后将计算的结果与字符串组合生成表名：“t_md5_val”。其中的 “t\*” 是固定的前缀，每个通过该映射关系自动生成的表都具有该前缀。
+子表名具体实现规则可以参考：[子表名生成规则代码实现](https://github.com/taosdata/TDengine/blob/06e05dfdf9c394e8ad5dd2ad4a54de53ea4e93f5/src/client/src/tscParseLineProtocol.c#L153-L188)。
```

Comment:
documentation should be more readable than code,  if not, we should improve the description in documentation instead of refering code